doc/keymap-format-text-v1.md

```diff
@@ -351,11 +351,98 @@ satisfied the LED is lit.
 
 ## The `xkb_symbols` section
 
```

Comment:
Since the symbols section has a bunch more statements which aren't described here, I'd like to add the following disclaimer for now:

```suggestion

NOTE: This section is incomplete.

```